app/src/utils.js

```diff
@@ -1,4 +1,5 @@
 const PERMISSION_WHITE_LIST = ['clipboard-read', 'notifications', 'openExternal'];
+export const WHITE_LISTED_DOMAIN = ['localhost', 'lisk.com'];
```

Comment:
This wont work as we have many applications which has blockchain application exploration? and they need to be redirected to?